### DISCUSSION TITLE
New blog having broken links and images

### DISCUSSION THREAD

**Discussion by mcewenkhundi** [quarto-dev/quarto-cli#755] [Q&A] [answered]
I have created a personal blog following the steps at https://quarto.org/docs/websites/website-blog.html that I have deployed with GitHub pages but it is having broken links in the posts and images. Here is the URL to my repository https://github.com/mcewenkhundi/mcewenkhundi would someone help to point me where I am wrong.

Thank you.

**Comment by dragonstyle** [comment]
I notice that a number of files and other things are missing from your `docs` folder (compare what is in the posts directories with what you see in the `_site` directory). I'm not sure how you're moving the files into `docs`, but I think however that is being done is perhaps not moving everything that needs to be moved.

You could try manually copying the contents of `_sites` to `docs`, letting that deploy, and then seeing whether all is well (which would confirm that it is just an issue generating the `docs` directory.

**Reply by mcewenkhundi** [reply]
Thank you for your quick response.

As to the question of adding files to the docs folder, I did not do that manually it was done automatically. I have copied the contents of _site to docs directory as you have suggested. The blog has improved but some images are still not rendered. I am using the default blog I haven't changed it.


Also when I was doing the initial git add .  I got the warnings in the attached image so maybe this is the source of the issue.
<img width="376" alt="giterror" src="https://user-images.githubusercontent.com/40823168/165823251-bcf76d14-2ed4-4cc3-b7d6-342c5833ba3e.PNG">

**Comment by dragonstyle** [comment]
(Ok the presence of the `output-dir` in your `_quarto.yml` file is directing us to write the site into the `docs` directory.)

Ok I just re-rendered the site locally and am taking a closer look. I'll post more here soon.

**Comment by dragonstyle** [comment]
Ok, I did find at least one issue with the way that we were forming the image src paths on windows (using the wrong separator!). I think that should get things going- I've started a fresh release which should be ready in a few minutes with the fix. Give that a try and let me know if that gets things going for you..

**Reply by mcewenkhundi** [reply]
What is the fresh release that you are referring to ? I understood this as the developer version of https://github.com/quarto-dev/quarto-cli , I did install it using the main branch following the instructions on the ReadMe. I recreated the blog repo but I still have the same issue.

Please help me understand how I can get the fresh release you are referring to.

**Reply by WillemSleegers** [reply]
@mcewenkhundi You can go to the GitHub page you linked and then check out the Releases section, there you can download the latest Preview.

**Comment by mcewenkhundi** [comment]
@WillemSleegers thanks for the pointer, I did go to the Releases section and managed to install the latest release using [quarto-0.9.324-win.msi]. I deleted and recreated my repo using the quarto blog project. This time when I rendered the blog preview in Rstudio it gave me the error below

Preparing to preview
[1/3] posts\post-with-code\index.qmd
Error running filter C:/Users/mkhundi/AppData/Local/Programs/Quarto/share/filters/rmarkdown/pagebreak.lua:
cannot open C:/Users/mkhundi/AppData/Local/Programs/Quarto/share/filters/rmarkdown/pagebreak.lua: No such file or directory

My session information.

R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Matrix products: default

locale:
[1] LC_COLLATE=English_United Kingdom.1252  LC_CTYPE=English_United Kingdom.1252    LC_MONETARY=English_United Kingdom.1252
[4] LC_NUMERIC=C                            LC_TIME=English_United Kingdom.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] compiler_4.1.0  fastmap_1.1.0   cli_3.2.0       htmltools_0.5.2 tools_4.1.0     yaml_2.2.1      rmarkdown_2.13 
 [8] knitr_1.37      xfun_0.30       digest_0.6.27   rlang_1.0.2     evaluate_0.14

**Reply by WillemSleegers** [reply]
Yeah I'm getting the same issue. I created an Issue about it: #759

**Reply by mcewenkhundi** [reply]
Thanks so much, hopefully it will be resolved.

**Comment by jjallaire** [comment, marked as answer]
This is now fixed, new release available here: https://github.com/quarto-dev/quarto-cli/releases/tag/v0.9.326

**Reply by mcewenkhundi** [reply]
Thanks so much, yes issue solved.
